### ISSUE TITLE
`.carousel-fade` and animated progress bar don't seem to use transition mixin

### ISSUE THREAD

**Issue by patrickhlauke** [twbs/bootstrap#25709]
Per https://github.com/twbs/bootstrap/pull/25641#issuecomment-367183255 is seems that even when disabling transitions (using `$enable-transitions` or after setting the "prefers reduced motion" option once that PR has landed), `.carousel-fade` (https://github.com/twbs/bootstrap/blob/3dd0bde664699fc7a191d0d3569f4f0ba8f06028/docs/4.0/components/carousel.md#crossfade) and the animated progress bar https://getbootstrap.com/docs/4.0/components/progress/#animated-stripes still use transitions. Possibly (not tested) even the JS-based `fade` https://getbootstrap.com/docs/4.0/components/list-group/#fade-effect

All these effects should be suppressed when `$enable-transitions` is off / "prefers reduced motion" is indicated by the user.

**Comment by mdo** [comment]
Given the referenced PR was merged, save to close?

**Comment by patrickhlauke** [comment]
No, the referenced PR addressed the reduced motion stuff which sets/unsets the the transitions variable. This issue points out that the carousel fade and animated progress bar are not conditional on that transitions variable (regardless of whether `$enable-transitions` is on or off / "prefers reduced motion" is set in the UA, carousels do animated fades and animated progress bars animate - they shouldn't/should obey those)
